vunit/ui.py

```diff
@@ -1023,6 +1023,9 @@ def get_implementation_subset(self, source_files):
         return SourceFileList([SourceFile(source_file, self._project, self)
                                for source_file in source_files])
 
+    def get_built_in_libraries(self):
```

Comment:
Functions on Public API should have documenation. Add documentation of the return value. Also since the function returns a set of names and not library object like other methods on VUnit class it should be named get_built_in_library_names.

Reply:
You added the documentation in the wrong place. It should be added on the VUnit method not the Builtins method. The Builtins method is not part of the public vunit api. You can keep the documentation of the Builtins method since it does not hurt but it should be also added to the VUnit class method.